### PR TITLE
LPS-198135 Create test for DDMTemplateVersionTableReferenceDefinition

### PR DESCRIPTION
**Pull request by gislayne-vitorino**
This is an update for:[ LPS-198135](https://liferay.atlassian.net/browse/LPS-198135)

### CODE REVIEW

**Comment by gislayne-vitorino** [comment]
ci:test:publications

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/change/tracking/test/DDMTemplateVersionTableReferenceDefinitionTest.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+		_ddmStructure = DDMStructureTestUtil.addStructure(
+			group.getGroupId(), DLFileEntryMetadata.class.getName());
+	}
+
+	@Override
+	protected CTModel<?> addCTModel() throws Exception {
+		return DDMTemplateTestUtil.addTemplate(
+			_ddmStructure.getStructureId(),
+			PortalUtil.getClassNameId(JournalArticle.class));
+	}
```
Comment (pei-jung) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/change/tracking/test/DDMTemplateVersionTableReferenceDefinitionTest.java:54]: Let's add the template in the setup and update it to get a new version.

**Review by pei-jung — APPROVED** [review]

**Comment by gislayne-vitorino** [comment]
ci:test:sf

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/change/tracking/test/DDMTemplateVersionTableReferenceDefinitionTest.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+			group.getGroupId(), DLFileEntryMetadata.class.getName());
+
+		_ddmForm = _ddmStructure.getDDMForm();
+	}
+
+	@Override
+	protected CTModel<?> addCTModel() throws Exception {
+		return _ddmStructureLocalService.updateStructure(
+			TestPropsValues.getUserId(), _ddmStructure.getStructureId(),
+			_ddmForm, _ddmStructure.getDDMFormLayout(),
```
Comment (pei-jung) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/change/tracking/test/DDMTemplateVersionTableReferenceDefinitionTest.java:57]: Inline _ddmStructure.getDDMForm() here. We don't need to declare a variable.

**Comment by gislayne-vitorino** [comment]
ci:test:sf

**Review by pei-jung — APPROVED** [review]

**Comment by pei-jung** [comment]
ci:forward

**Comment by david-truong** [comment]
ci:forward

**Comment by david-truong** [comment]
ci:forward:force

**Comment by david-truong** [comment]
ci:forward:force

**Pull request closed without merging**